src/Rings/mpolyquo-localizations.jl

```diff
@@ -219,7 +219,6 @@ function Base.show(io::IO, ::MIME"text/plain", L::MPolyQuoLocRing)
 end
 
 function Base.show(io::IO, L::MPolyQuoLocRing)
-  io = pretty(io)
   if get(io, :supercompact, false)
     print(io, "Localized quotient of multivariate polynomial ring")
   else
```

Comment:
RIght below this line, there is a similar thing wrong with the order of calling `pretty` and setting `supercompact => true` as the one already fixed for `MPolyLeadingMonOne`. The doctest failure seems related to this.

Reply:
I fixed the doctest, but I couldn't find the spot you are talking about (I think?). If you still see an issue, please pinpoint me. Thanks!

Reply:
You already fixed it